routing/routing_integration_tests/routing_integration_tests.pro

```diff
@@ -11,7 +11,7 @@ CONFIG -= app_bundle
 TEMPLATE = app
 
 ROOT_DIR = ../..
-DEPENDENCIES = map routing search storage indexer platform geometry coding base osrm jansson protobuf tomcrypt succinct stats_client
+DEPENDENCIES = map routing search storage indexer platform geometry coding base osrm jansson protobuf tomcrypt succinct stats_client opening_hours
```

Comment:
Any reason why the opening_hours target was added to the routing integration tests?